usaspending_api/urls.py

```diff
@@ -23,7 +23,9 @@
 
 urlpatterns = [
     url(r'^status/', views.StatusView.as_view()),
+    url(r'^status2/', views.StatusView.as_view()),
     url(r'^api/v1/federal_accounts/', include('usaspending_api.accounts.urls_federal_account')),
+    url(r'^api/v2/federal_obligations/', include('usaspending_api.accounts.urls_federal_obligations')),
```

Comment:
If we're already fixing other things in this file, can you group all the v2 endpoints under the v1 endpoints? ideally we'd want them grouped and alphabetized within their groups.